src/main/scala/io/iohk/ethereum/jsonrpc/EthService.scala

```diff
@@ -264,6 +276,42 @@ class EthService(
     }
   }
 
+  def getCode(req: GetCodeRequest): ServiceResponse[GetCodeResponse] = {
+    Future.successful {
+      resolveBlock(req.block).map { block =>
+        val world = InMemoryWorldStateProxy(blockchainStorages, Some(block.header.stateRoot))
```

Comment:
You are right, it doesn't cause any additional overhead. So I think we can leave as it is :+1: